sample/src/main/java/com/mapzen/android/sample/BasicMapzenActivity.java

```diff
@@ -48,10 +50,15 @@
     final boolean enabled = (savedInstanceState == null ||
         savedInstanceState.getBoolean(KEY_LOCATION_ENABLED));
 
+    final float millis = System.currentTimeMillis();
+
     final MapFragment mapFragment =
         (MapFragment) getSupportFragmentManager().findFragmentById(R.id.fragment);
     mapFragment.getMapAsync(new OnMapReadyCallback() {
       @Override public void onMapReady(MapzenMap map) {
+        float now = System.currentTimeMillis();
+        float elapsedTime = now - millis;
```

Comment:
System time should be captured in a `long` instead of a `float`. Otherwise the difference is lost in precision and `0.0` elapsed time is always reported.

Ditto for local vars `now` and `millis` above.